addons/web/static/tests/views/kanban_tests.js

```diff
@@ -7528,6 +7528,38 @@ QUnit.module('Views', {
 
         kanban.destroy();
     });
-});
 
+    QUnit.test("Instantiate widget with option parameters", async function (assert) {
+        assert.expect(2);
+
+        widgetRegistry.add('optionwidget', Widget.extend({
+            init(parent, state, options) {
+                this._super(...arguments);
+                this.title = options.attrs.title;
+            },
+            start() {
+                this.$el.html(`<div class="option-widget">${_.escape(this.title)}</div>`);
```

Comment:
```js
$("<div>", {text: this.title, class: 'option-widget'}));
```
doesn't work btw?

Reply:
It should work fine, it's just little known and usually forgotten about.